discord_bots/main.py

```diff
@@ -189,20 +188,27 @@ async def after_invoke(context: Context):
     context.session.close()
 
 
-def main():
-    load_dotenv()
+async def setup():
```

Comment:
really appreciate all the code cleanups you're including! looks so much better

Reply:
some necessary refactors for supporting discord.py >2.0 
https://discordpy.readthedocs.io/en/stable/migrating.html#extension-and-cog-loading-unloading-is-now-asynchronous